server/test/controllers/auth/createUser.test.js

```diff
@@ -0,0 +1,42 @@
+import { expect, vi, describe, it } from "vitest";
+import createUserRequestHandler from "../../../src/controllers/auth/createUser";
+
+describe("createUser test suite", () => {
```

Comment:
We could probably add a couple more tests
1. Sign-In (Success and Failure)
2. Register (Success and Failure)

I can work on getting a coverage library installed and then I can go back and add any remaining tests after you merge this in

Reply:
Alright, sounds good!!